src/aktualizr_secondary/secondary_tcp_server.cc

```diff
@@ -159,6 +167,7 @@ bool SecondaryTcpServer::HandleOneConnection(int socket) {
         if (install_result == data::ResultCode::Numeric::kNeedCompletion) {
           need_reboot = true;
         }
+        LOG_TRACE << "Install complete " << (need_reboot ? "but need reboot" : "!");
```

Comment:
```suggestion
        LOG_INFO << "Installation complete" << (need_reboot ? ", but needs reboot to complete." : "!");
```
And again, can we also print something about the Target, like the name and version?